tests/src/test/scala/ch/epfl/bluebrain/nexus/tests/kg/VersionSpec.scala

```diff
@@ -0,0 +1,70 @@
+package ch.epfl.bluebrain.nexus.tests.kg
+
+import akka.http.scaladsl.model.{StatusCodes, Uri}
+import ch.epfl.bluebrain.nexus.testkit.EitherValuable
+import ch.epfl.bluebrain.nexus.tests.Tags.VersionTag
+import ch.epfl.bluebrain.nexus.tests.iam.types.Permission
+import ch.epfl.bluebrain.nexus.tests.kg.VersionSpec.VersionBundle
+import ch.epfl.bluebrain.nexus.tests.{BaseSpec, Identity}
+import io.circe.generic.semiauto.deriveDecoder
+import io.circe.{Decoder, Json}
+import monix.execution.Scheduler.Implicits.global
+
+class VersionSpec extends BaseSpec with EitherValuable {
+
+  val versionUri: Uri = Uri(s"http://${System.getProperty("delta:8080")}/v1/version")
+
+  "The /version endpoint" should {
+    s"be protected by ${Permission.Version.Read.value}" in {
+      deltaClient.get[Json]("/version", Identity.Anonymous) { (_, response) =>
+        response.status shouldEqual StatusCodes.Forbidden
+      }
+    }
+
+    "return the dependencies and plugin versions" taggedAs VersionTag in {
+      aclDsl.addPermissionAnonymous("/", Permission.Version.Read).runSyncUnsafe()
+
+      deltaClient.get[Json]("/version", Identity.Anonymous) { (json, response) =>
+        response.status shouldEqual StatusCodes.OK
+        json.as[VersionBundle].rightValue
```

Comment:
Usually in integration tests we should not have any circe magic. Just compare with an actual file or template file. It is easier to debug and easier to avoid circe errors.

Reply:
The versions differ, we can't compare to a file or template...

```
{
  "@context": "https://bluebrain.github.io/nexus/contexts/version.json",
  "delta": "1.4.1+310-2d8740d4+20210413-1214",
  "dependencies": {
    "blazegraph": "2.1.5",
    "cassandra": "3.11.8",
    "elasticsearch": "7.12.0",
    "remoteStorage": "1.4.1+310-2d8740d4"
  },
  "plugins": {
    "archive": "1.4.1+310-2d8740d4+20210413-1214",
    "blazegraph": "1.4.1+310-2d8740d4+20210413-1214",
    "composite-views": "1.4.1+310-2d8740d4+20210413-1214",
    "elasticsearch": "1.4.1+310-2d8740d4+20210413-1214",
    "storage": "1.4.1+310-2d8740d4+20210413-1214"
  }
}
```

Reply:
ok